examples/web_app/src/document.rs

```diff
@@ -13,7 +13,7 @@ pub struct Crate {
     version: String,
 }
 
-// Implement the Document trait so that we can use our struct with Meilisearch
+// Implement the IndexConfig trait so that we can use our struct with Meilisearch
```

Comment:
I think this documentation may be outdated. We do not implement the indexConfig trait here.